megazords/ios-rust/MozillaRustComponents.h

```diff
@@ -18,3 +18,4 @@
 #import "placesFFI.h"
 // This is the ffi header and will be deleted after uniffi
 #import "RustPlacesAPI.h"
+#import "GleanFfi.h"
```

Comment:
Should this be `glean.h`? From https://github.com/mozilla/application-services/pull/4619/files#diff-3729d0e227487c58350ead0b2d76baa018fcca7439fb97e5ec1c2b4620fd1ee1R136, or is this different?

Reply:
I believe I am following similar to how we were bundling our builds with Carthage on the `megazords/ios` side of things https://github.com/mozilla/application-services/blob/main/megazords/ios/MozillaAppServices.h#L18